packages/integrations/markdoc/src/extensions/html/htmlattributes/mutate-raw-html-attributes-to-react-compatible-variants.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Given a Record<string, string> of raw HTML attributes, fix various cases to be React
```

Comment:
Are React-specific transforms necessary for our Astro x Markdoc renderer? I'd be curious if tests continue to pass without this 🤷

Reply:
Added test coverage of randomly cased HTML attributes in Markdoc markup to ensure this was not problematic